airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/state_converters/datetime_stream_state_converter.py

```diff
@@ -81,26 +82,34 @@ def convert_from_sequential_state(self, cursor_field: CursorField, stream_state:
         {
             "state_type": ConcurrencyCompatibleStateType.date_range.value,
             "metadata": { … },
+            "start": <timestamp representing the latest date before which all records were synced>
```

Comment:
Do we need to set this as part of the state? Could it simply be a field in `DateTimeStreamStateConverter`? I'm wary of adding stuff in the state as it means possible breaking changes when we want to change that

Reply:
Ok! I think I have more context given your comment. 

In my mind, the `_state` was something we could emit when we will move from the sequential state to the new fancy concurrent state hence why I was surprised. Will having `start` as part of the response from `convert_from_sequential_state` create more work than having an internal field when we will do this switch?